packages/twenty-server/src/modules/messaging/message-import-manager/services/messaging-import-exception-handler.service.ts

```diff
@@ -25,7 +25,6 @@ import {
 } from 'src/modules/messaging/message-import-manager/exceptions/message-import.exception';
 
 export enum MessageImportSyncStep {
-  FULL_MESSAGE_LIST_FETCH = 'FULL_MESSAGE_LIST_FETCH', // TODO: deprecate to only use MESSAGE_LIST_FETCH
```

Comment:
let's remove it as we were not handling both MESSAGE_LIST_FETCH and FULL_MESSAGE_LIST_FETCH